src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,16 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	public static final String SKRIPT_PREFIX = "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
+
+	public static String getSkriptPrefix() {
+		return SKRIPT_PREFIX_MESSAGE.getValueOrDefault(SKRIPT_PREFIX);
+	}
```

Comment:
There doesn't need to be a method for getting the prefix. You should be statically access the field constant, no one is going to need to get the prefix of Skript outside of Skript.

Reply:
The getter is justified here, it reduces duplicate code and protects the `SKRIPT_PREFIX_MESSAGE` field.

Reply:
It's already final, there is no protection.